packages/react-next/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -70,7 +70,7 @@ export const ChoiceGroupOptionBase = (props: IChoiceGroupOptionProps) => {
         {imageSrc && (
           <div className={classNames.innerField}>
             <div className={classNames.imageWrapper}>
-              <Image src={selectedImageSrc} alt={imageAlt} {...imageSize} />
+              <Image src={imageSrc} alt={imageAlt} {...imageSize} />
```

Comment:
Why this change? Bug fix?

Reply:
It is an intended bug fix that screener noticed in this pr. `ChoiceGroupOption`’s is supposed to have an Image src for both imageSrc and selectedImageSrc props to allow for a unique selection image. Here's what the fix does:

Without Fix:
![image](https://user-images.githubusercontent.com/7192338/93548242-b2f42780-f91b-11ea-80a1-6bf2145f0cbe.png)

Fix:
![image](https://user-images.githubusercontent.com/7192338/93548263-bd162600-f91b-11ea-9c5c-1cb646800d85.png)